File: Data Structures & Algorithms/valid-palindrome-ii/submission-1.py
class Solution:
    def validPalindrome(self, s: str) -> bool:
        i = 0
        j = len(s) - 1
        count = 0
        while i < j:
            if s[i] != s[j]:
                count +=1
            i += 1
            j -= 1
        return True if count <= 0 else False

File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
  unordered_map<int,int> ns;

  for(int i = 0; i < nums.size(); i++){
    if(ns.count(target - nums[i])){
      return {ns[target - nums[i]], i};
    }
    ns[nums[i]] = i;
  }

  return {};

};
